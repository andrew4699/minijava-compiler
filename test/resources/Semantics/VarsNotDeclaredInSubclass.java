class Main {
  public static void main(String[] args) {
    System.out.println(new B().kerchoo());
  }
}

class A {
  int r;

  public int foo(int a, int b, int c) {
    return 5;
  }
}

class B extends A {
  public int kerchoo() {
    hi = true;
    r = r + 10;
    hi = false;
    return 5;
  }
}
