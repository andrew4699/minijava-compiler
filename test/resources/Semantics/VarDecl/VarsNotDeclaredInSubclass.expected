[VarDeclVisitor - 17]: hi undeclared at B.kerchoo
[VarDeclVisitor - 17]: hi undeclared at B.kerchoo
[VarDeclVisitor - 19]: hi undeclared at B.kerchoo
[VarDeclVisitor - 19]: hi undeclared at B.kerchoo
